classes/classes/Items/MutationsHelper.as

```diff
@@ -53,6 +53,33 @@ package classes.Items
 			return rand(Math.min(min + changes, max)) === 0;
 		}
 
+		/**
+		 * Initializes the transformation. Meaning: changes is set to 0 and the initial changeLimit is being determined.
+		 * @param   rolls             An array of the rolls to randomly increase the changeLimit. e. g.: [2, 3, 4]
+		 * @param   startChangeLimit  The initial changeLimit
+		 * @param   minChangeLimit    The minimum changeLimit
```

Comment:
Please add more detailed documentation, code is read more often than written.

e.g.:
Initializes the transformation. Meaning: changes is set to 0 and the initial changeLimit is being determined.
This is done by performing rolls, applying modifiers from Perks and enforcing the minimum number of changes.

`startChangeLimit` The initial amount of changes to start with before rolls or perks
`minChangeLimit` The enforced minimum amount of changes, i.e. at least this many changes will be performed

Reply:
Ok, BTDT.
But actually the `changeLimit` is the maximum amount of changes that may happen. If the player is unlucky or there's nothing to be changed he may still end up with 0 `changes` no matter how high the `changeLimit` is.